.github/workflows/publish.yml

```diff
@@ -92,6 +92,15 @@ jobs:
             latest=auto
             prefix=
             suffix=-${{ matrix.ubuntu_version }}-${{ matrix.arch }}
+          tags: |
+            # Generate old tag names (e.g main-s390x, main-x86_64...) when building focal
+            # branch event
+            type=ref,enable=${{ matrix.ubuntu_version == 'focal' }},suffix=-${{ matrix.arch }},event=branch
+            # pr event
+            type=ref,enable=${{ matrix.ubuntu_version == 'focal' }},prefix=pr-,suffix=-${{ matrix.arch }},event=pr
+            # tags for all pr/branches
+            type=ref,event=branch,enable=true,priority=600
+            type=ref,event=pr,enable=true,prefix=pr-,priority=600          
```

Comment:
According to https://github.com/docker/metadata-action?tab=readme-ov-file#priority-attribute, everything here has priority 600? Unclear to me what will happen if multiple are present. Maybe give them unique values?

Reply:
I actually don't think we use priority in our case. Once we have transitioned to the new tag, this will be stripped out anyway.